pkg/usermanagement/client.go

```diff
@@ -2334,3 +2378,48 @@ func (c *Client) RevokeSession(ctx context.Context, opts RevokeSessionOpts) erro
 
 	return nil
 }
+
+func (c *Client) ListSessions(ctx context.Context, opts ListSessionsOpts) (ListSessionsResponse, error) {
+	req, err := http.NewRequest(
+		http.MethodGet,
+		fmt.Sprintf("%s/user_management/sessions", c.Endpoint),
```

Comment:
The list sessions endpoint is scoped to a particular user, so this should be something like:
```suggestion
		fmt.Sprintf("%s/user_management/users/%s/sessions", c.Endpoint, <userID field>),
```